website/templates/emails/fork_completed.html.mako

```diff
@@ -6,7 +6,7 @@
 <tr>
   <td style="border-collapse: collapse;">
 
-    <h3 class="text-center" style="padding: 0;margin: 30px 0 0 0;border: none;list-style: none;font-weight: 300;text-align: center;">The fork of <b>${title}</b> has been successfully created: <a href="${settings.DOMAIN + guid}">here.</a></h3>
+    <h3 class="text-center" style="padding: 0;margin: 30px 0 0 0;border: none;list-style: none;font-weight: 300;text-align: center;">The fork of <b>${title}</b> has been successfully created here: ${settings.DOMAIN + guid}</h3>
```

Comment:
Do email clients and sites automatically parse urls even on html emails, or should these still be links?

Reply:
They seem to for plain text, but I found a counter example in prereg_challenge email so I'll change this.